File: CSharp/DependencyBreakingTests/C/CheckoutTest.cs
using Org.Codecop.Dependencies.C;
using Xunit;

namespace Org.Codecop.Dependencies.C.Tests
{
    public class CheckoutTest
    {
        [Fact]
        public void Test3()
        {
            var checkout = new Checkout();
            checkout.CreateReceipt(new Money(12));
            Assert.NotNull(checkout);
        }
    }
}

File: PHP/test/C/CheckoutTest.php
<?php

namespace Org\Codecop\Dependencies\Tests\C;

use PHPUnit\Framework\TestCase;
use Org\Codecop\Dependencies\C\Checkout;
use Org\Codecop\Dependencies\C\Money;

class CheckoutTest extends TestCase {

    public function test3() {
        $checkout = new Checkout();

        $checkout->createReceipt(new Money(12));

        $this->assertNotNull($checkout);
    }
}
